sonic-chassisd/scripts/chassisd

```diff
@@ -302,16 +313,20 @@ class ModuleUpdater(logger.Logger):
                     elif prev_status != ModuleBase.MODULE_STATUS_ONLINE:
                         self.log_notice("Module {} is on-line!".format(key))
 
-                for asic_id, asic in enumerate(module_info_dict[CHASSIS_MODULE_INFO_ASICS]):
-                    asic_global_id, asic_pci_addr = asic
-                    asic_key = "%s%s" % (CHASSIS_ASIC, asic_global_id)
-                    if not self._is_supervisor():
-                        asic_key = "%s|%s" % (key, asic_key)
+                    module_cfg_status = self.get_module_admin_status(key)
+
+                    #Only populate the related tables when the module configure is up
+                    if module_cfg_status != 'down':
```

Comment:
When the supervisor boots up, do we assume the module_admin_status is default 'UP' ?

Reply:
By default it is UP.  Which means the configuration doesn't contain related Module config CHASSIS_MODULE in CONFIG_DB.  If configuration contains CHASSIS_MODULE entry as below, in most of the cases, the admin_status value should be "down" ( Could be "up" too,. but the currently design is to delete this entry when user issues  "config chassis module startup <module_name>"). 

If configuration contains the following:
```
    "CHASSIS_MODULE": {
        "FABRIC-CARD3": {
            "admin_status": "down"
        }
    },
```